debian/changelog

```diff
@@ -1,3 +1,50 @@
+snapcraft (2.10) xenial; urgency=medium
+
+  [ Martin Wimpress ]
+  * Correct autotools tests to use configflags (#521)
+
+  [ Leo Arias ]
+  * Run the integration tests against a local fake server when the user
+    password is not in the environment. (#511) (LP: #1585023)
+  * Move the login and logout methods to a client. (#518) (LP: #1586504)
+  * Improve the config handling. (#519) (LP: #1586511)
+  * Fix the one-time password login. (#529) (LP: #1586832)
+  * Moved the download to the store client. (#530) (LP: #1586836)
+  * Moved the upload to the store client. (#531) (LP: #1586836)
+  * Updated the documentation about the icon. (#542) (LP: #1578231)
+  * Improve the error message when a part binary is not found. (#541)
+    (LP: #1582367)
+  * Try running the full suite. (#520) (LP: #1588098)
```

Comment:
Maybe this message can be changed to: Remove the skip of the ROS example in autopkgtest.